src/rule-generated-flow-types.js

```diff
@@ -356,7 +356,46 @@ module.exports = {
         }
         useFragmentInstances.push({
           fragmentName: fragmentName,
-          node: node
+          node: node,
+          hookName: 'useFragment'
+        });
+      },
+
+      /**
+       * usePaginationFragment() calls
+       */
+      'CallExpression[callee.name=usePaginationFragment]'(node) {
+        const firstArg = node.arguments[0];
```

Comment:
Should we reduce the duplication? 2 ideas:

- Combine the "css selectors" if that's supported and look for `node.callee.name` for the error message.
- Factor out the body into something like `checkHook(node, 'usePaginationFragment')`?

Reply:
@kassens they're such small functions that I figured it wasn't that much worth it. If you feel strongly I can update it